# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/m1n1/hotfix-sh.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- m1n1/version.sh.vanilla	2023-02-21 18:56:48.081348231 +0100
+++ m1n1/version.sh	2023-02-21 18:56:55.459348821 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 cd "$(dirname "$0")"
 
 dirbase="$(basename "$(pwd)")"
